src/protocols/BaseProtocol.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ *  Represents common attributes and behavior of a protocol
+ */
+export abstract class BaseProtocol {
+  static title: string;
+
+  static symbol: string;
+
+  static derivationPath: string;
```

Comment:
I would keep this on sdk side and add a method to create an account instead.

Reply:
I kept it here so that I can achieve this https://github.com/aeternity/superhero-wallet/pull/2219/files#diff-629f43758798f7d5446cb5e8fc37a20d33204b606127a338f28924376b805d71R22
by using existing methods from the SDK to generate private keys for Bitcoin.

Reply:
> existing methods from the SDK to generate private keys for Bitcoin

aepp-sdk-js can't do it https://github.com/aeternity/superhero-wallet/pull/2219#discussion_r1286085523

Reply:
Yes. Static properties have been removed. We will use the methods from each protocol SDK to generate accounts.